trunk/SUAVE/Methods/Geometry/Three_Dimensional/angles_to_dcms.py

```diff
@@ -173,68 +173,4 @@ def new_tensor(a):
     
     T = np.resize(T,[n_a,3,3])
     
-    return T
```

Comment:
You should make this a regression

Reply:
So this is already being covered...are you sure you want another regression?

Reply:
No leave it alone